### ISSUE TITLE
Seems the module doesn't work out of the box? 

### ISSUE THREAD

**Issue by darrenchiu** [ascoders/react-native-image-viewer#27]
I have just followed the instruction from readme but getting the following error:
```

Warning: Failed prop type: Invalid prop `source` supplied to `Image`.
    in Image (created by ImageViewer)
    in ImageViewer (created by Transmit)
    in Transmit (created by myproject)
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer (created by Modal)
    in RCTView (created by View)
    in View (created by Modal)
    in RCTModalHostView (created by Modal)
    in Modal (created by myproject)
    in myproject
    in RCTView (created by View)
    in View (created by AppContainer)
    in RCTView (created by View)
    in View (created by AppContainer)
    in AppContainer


RCTJSONStringify() encountered the following error: Invalid type in JSON write (NSURL)
```

Is that because my react version is too new? I have tried with 0.39 and 0.41, both failing. Thanks in advance

**Comment by ascoders** [comment]
Invalid prop \`source\` supplied to \`Image\` seems like you set error source.

I saw the latest version of the `image component`, source field has not been modified, it seems like you set wrong url field.

**Comment by stakkdarren** [comment]
i just use the same as those in the example...

**Comment by darrenchiu** [comment]
i just copy the same source from the example

**Comment by usejony** [comment]
i have error same of you

**Comment by Micjoyce** [comment]
+1
has anyone fix it?

**Comment by fdjiangwu** [comment]
i have error same of you, i just copy the same source from the example

**Comment by ascoders** [comment]
That's true. I'm looking for reasons.

**Comment by ascoders** [comment]
Well, don't worry, because the link address protocol in demo is blocked. There's no problem with a new address:

```typescript
const images = [{
    url: 'https://avatars2.githubusercontent.com/u/7970947?v=3&s=460'
}, {
    url: 'https://avatars2.githubusercontent.com/u/7970947?v=3&s=460'
}, {
    url: 'https://avatars2.githubusercontent.com/u/7970947?v=3&s=460'
}]
```

**Comment by faogustavo** [comment]
It was solved? I tried a Base64 with RN 0.42 and still not work. Direct on the ImageView it works :/

**Comment by ascoders** [comment]
@fdjiangwu Try to preset width and height? https://github.com/ascoders/react-native-image-viewer/blob/master/image-viewer/image-viewer.component.tsx#L136

```javascript
const images = [{
    url: 'data:image/png;base64,.....',
    width: 200,
    height: 200
}]
```
